civihr_employee_portal/views/civihr_employee_portal.views.inc

```diff
@@ -1117,29 +1117,61 @@ function removeAdditionalCommas($view) {
   return $fieldsToClean;
 }
 
-function removeInactiveDepartments($results = array()) {
-  foreach ($results as $key => $value) {
-    $startDate = !empty($value->hrjc_role_civicrm_contact_start_date) ? $value->hrjc_role_civicrm_contact_start_date : NULL;
-    $endDate = !empty($value->hrjc_role_civicrm_contact_end_date) ? $value->hrjc_role_civicrm_contact_end_date : NULL;
+/**
+ * Sets the name to NULL for relationships starting in future or ending in past
+ *
+ * @param array $results
+ * @return array
+ */
+function removeInactiveManagers($results = []) {
+  $endField = 'civicrm_relationship_end_date';
+  $startField = 'civicrm_relationship_civicrm_contact_1_start_date';
+  $startOfToday = new \DateTime('today 00:00:00');
+  $endOfToday = new \DateTime('today 23:59:59');
+
+  $startDates = [];
 
-    if (($endDate && $endDate < date('Y-m-d h:i:s') && $endDate != "0000-00-00 00:00:00") ||
-            ($startDate && $startDate > date('Y-m-d h:i:s'))
-    ) {
-      $value->hrjc_role_civicrm_contact_department = NULL;
+  foreach ($results as $key => $value) {
+    $end = $value->$endField ? new \DateTime($value->$endField) : NULL;
+    $start = $value->$startField ? new \DateTime($value->$startField) : NULL;
+    $startDates[] = $start;
+    $isActive = $value->civicrm_relationship_is_active == 1;
+    $isPast = $end && $end < $startOfToday;
+    $isFuture = $start > $endOfToday;
+
+    if (!$isActive || $isPast || $isFuture) {
+      unset($results[$key]);
     }
   }
+
   return $results;
 }
 
-function removeInactiveManagers($results = array()) {
-  foreach ($results as $key => $value) {
-    if ($value->civicrm_relationship_is_active == 0 ||
-            ($value->civicrm_relationship_end_date < date('Y-m-d') && $value->civicrm_relationship_end_date != NULL) ||
-            $value->civicrm_relationship_start_date > date('Y-m-d')) {
-      $value->civicrm_contact_civicrm_relationship_first_name = NULL;
-      $value->civicrm_contact_civicrm_relationship_last_name = NULL;
+/**
+ * @param \stdClass[] $results
```

Comment:
even though both `removeInactiveRoleDetails` and `removeInactiveManagers` receive the same param, the param type on the their docblocks is different. Please use the same for both.

Please also add a description to this function